<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>179</width>
      <height>412</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="Draw_but_3">
       <property name="text">
        <string>Отрисовать график</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clear_But_3">
       <property name="text">
        <string>Очистить экран</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Show_dick">
       <property name="text">
        <string>Вывести таблицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Draw_From_But_3">
       <property name="text">
        <string>Пойти нахуй</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Exit_But_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Закрыть программу</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>X=[</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="X_EDIT_START_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="X_EDIT_END_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Точность</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Acc_Edit_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Начальное значение y и y'</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Start_Posit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Start_y_for_sys">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Шаг по пространству</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="StepEdit_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Get_Data_but_3">
         <property name="text">
          <string>Получить данные</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>440</y>
      <width>196</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="Test_task_button">
       <property name="text">
        <string>Пробная задача</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="main_task">
       <property name="text">
        <string>Основная задача №1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="main_task_syst">
       <property name="text">
        <string>Основная задача№2(сист)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>501</width>
      <height>761</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1330</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
